resources/views/blade/input/text.blade.php

```diff
@@ -0,0 +1,8 @@
+@props([
+    'item' => null,
+    'field_name' => null,
+    'input_style' => null,
+    'required' => false,
+])
+<!-- input-text blade component -->
+<input {{ $attributes->merge(['class' => 'form-control']) }} {{ ($input_style) ? $attributes->merge(['style' => $input_style]): '' }} {{ ($required ?? false) ? 'required' : '' }} />
```

Comment:
Readability can be improved here as well by combining the `$attribute->merge()` calls and using the `@required` helper.


```suggestion
<input
    {{ $attributes->merge(['class' => 'form-control', 'style' => $input_style]) }}
    @required($required)
/>
```